lib/pages/settings/view/settings_page.dart

```diff
@@ -23,6 +24,19 @@ class SettingsPage extends StatefulWidget {
 }
 
 class _SettingsPageState extends State<SettingsPage> {
+  PackageInfo _packageInfo = PackageInfo(
+    version: 'Unknown',
+  );
+
+  Future<void> _initPackageInfo() async {
+    if (!Platform.environment.containsKey('FLUTTER_TEST')) {
```

Comment:
Add a comment mentioning why this is necessary (and a link to the issue you create).